scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulerFrontend.java

```diff
@@ -409,6 +407,31 @@ public TaskResult getTaskResult(String jobId, String taskName) throws NotConnect
         return this.getTaskResult(JobIdImpl.makeJobId(jobId), taskName);
     }
 
+
+    @Override
+    public List<TaskResult> getTaskResultByTag(JobId jobId, String taskTag)
```

Comment:
I agree with Laurent, other similar methods in the SchedulerFrontend are tagged as @ImmediateService